students/preactive/Lesson2/fizzbuzz.py

```diff
@@ -0,0 +1,10 @@
+for i in range(1,100):
+    currentprint = str(i)
+    if i % 3 == 0:
+        currentprint = currentprint.replace(str(i),"")
+        currentprint += "fizz"
+    if i % 5 == 0:
+        currentprint = currentprint.replace(str(i),"")
+        currentprint += "buzz"
+    print(currentprint)
+    currentprint = ""
```

Comment:
This last line is not necessary since you're resetting the `current print` value at the beginning of the next iteration

Reply:
removed line 10